### PR TITLE
Automating Furnaces and Blast Furnaces

### PR DESCRIPTION
**Pull request by Rakkis157**
Some recipes to fill in some holes in automation:

Furnace Assembler Recipe, for Bronze/Steel Boilers and Furnaces.
Blast Furnace Assembler Recipe, for SBF and EBF.
Blast Furnace Steel Plate crafting recipe.

### CODE REVIEW

File: src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/furnace.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "type": "modern_industrialization:assembler",
```
Comment (Azercoco) [inline src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/furnace.json:2]: I would make this recipe in the packer instead

File: src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/furnace.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "eu": 8,
+  "duration": 200,
+  "item_inputs": [
+    {
+      "tag": "minecraft:cobblestone",
```
Comment (Azercoco) [inline src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/furnace.json:7]: the tag is #minecraft:stone_crafting_materials

**Review by Azercoco — CHANGES_REQUESTED** [review]
Just apply the two little fixes and I will merge it. Thanks

**Review by Azercoco — APPROVED** [review]
Ok this is good ! Thank you ! Maybe we should add Steam Blast Furnace recipe in the packer ?

**Comment by Rakkis157** [comment]
> Ok this is good ! Thank you ! Maybe we should add Steam Blast Furnace recipe in the packer ?

And the Coke Oven as well. I'll do that real quick.

**Comment by Rakkis157** [comment]
It would be odd to have the blast furnace be an assembler recipe and the SBF be in the packer. Should I move the blast furnace to the packer as well, or have the SBF be in the assembler?

**Comment by Azercoco** [comment]
> It would be odd to have the blast furnace be an assembler recipe and the SBF be in the packer. Should I move the blast furnace to the packer as well, or have the SBF be in the assembler?

No having the SBF in the packer is ok because it's in steam age where you don't have access to the assembler. It would not make sense to have the SBF in the assembler because when you unlock the latter, you have no use of the former anymore.

**Comment by Rakkis157** [comment]
> > It would be odd to have the blast furnace be an assembler recipe and the SBF be in the packer. Should I move the blast furnace to the packer as well, or have the SBF be in the assembler?
> 
> No having the SBF in the packer is ok because it's in steam age where you don't have access to the assembler. It would not make sense to have the SBF in the assembler because when you unlock the latter, you have no use of the former anymore.

Also I was talking about moving the vanilla Blast Furnace to the packer. Should I do so?

**Comment by Azercoco** [comment]
> > > It would be odd to have the blast furnace be an assembler recipe and the SBF be in the packer. Should I move the blast furnace to the packer as well, or have the SBF be in the assembler?
> > 
> > 
> > No having the SBF in the packer is ok because it's in steam age where you don't have access to the assembler. It would not make sense to have the SBF in the assembler because when you unlock the latter, you have no use of the former anymore.
> 
> Also I was talking about moving the vanilla Blast Furnace to the packer. Should I do so?

No I prefer that most electric age recipe stay in the assembler

**Review by Azercoco — APPROVED** [review]
Well done, and nice catch for the missing amethyst block in the packer !

**Pull request merged**